doc/man/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@ add_custom_command(
 )
 
 install(
-	FILES ${CMAKE_CURRENT_BINARY_DIR}/iotrace.8.gz
+	FILES ${CMAKE_CURRENT_SOURCE_DIR}/MAN.md
```

Comment:
We decided to remove MAN

Reply:
This line here was mistakingly changed. I will revert it.
I kept man in normal "make install" installation process.
The package does not install man.